generators/uscore-r4/generator.rb

```diff
@@ -10,6 +10,19 @@
 OUT_PATH = '../../lib/app/modules'
 RESOURCE_PATH = '../../resources/us_core_r4/'
 
+PROFILE_URIS = {
+  smoking_status: 'https://build.fhir.org/ig/HL7/US-Core-R4/StructureDefinition-us-core-smokingstatus.json',
```

Comment:
Why are different urls used here and in `ValidationUtil`?

Reply:
These are the profile URI's from the build version of US Core which the generator uses. The ValidationUtil checks against the published version.

Reply:
But why are different versions used?

Reply:
I don't think the published version existed when I started the generator. It looks like it was made on June 19.

Reply:
But they exist now, so what purpose do these urls serve?